INSTALL.md

```diff
@@ -26,33 +25,41 @@ source code.  Unpack as follows.
     tar xzf <some dir>/accumulo-X.Y.Z-bin.tar.gz
     cd accumulo-X.Y.Z
 
+There are four scripts in the the `bin/` directory that are use to manage Accumulo:
+
+1. `accumulo` - Runs Accumulo command-line tools and starts Accumulo processes
+2. `accumulo-service` - Runs Accumulo processes as services
+3. `accumulo-single`- Manages Accumulo on a single node
+4. `accumulo-cluster` - Manages Accumulo on a cluster
```

Comment:
> It's pretty easy to create them using accumulo-cluster bootstrap-config and it would be nice to reduce the number of scripts in bin/.

+1. Automating the creation of configuration files for a single-node installation sounds like the right way to approach this.